extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -14,6 +26,63 @@ export class AmpStoryShoppingAttachment extends AmpStoryPageAttachment {
 
     /** @private {?Element} */
     this.container_ = null;
+
+    /** @private @const {?../../amp-story/1.0/amp-story-store-service.AmpStoryStoreService} */
+    this.storeService_ = null;
+  }
+
+  /**
+   * @private
+   * @return {?Element} the parent amp-story-page
+   */
+  getPage_() {
+    return closest(
+      dev().assertElement(this.element),
+      (el) => el.tagName.toLowerCase() === 'amp-story-page'
+    );
+  }
+
+  /**
+   * Updates the CTA based on the shopping data.
+   * @param {!ShoppingDataDef} shoppingData
+   * @private
+   */
+  updateCtaText_(shoppingData) {
+    const pageIdElem = this.getPage_();
+
+    const shoppingTags = Array.from(
+      pageIdElem.getElementsByTagName('amp-story-shopping-tag')
+    ).filter((tag) => shoppingData[tag.getAttribute('data-tag-id')]);
+
+    const ctaButton = pageIdElem
```

Comment:
This variable name is a little confusing because it's a reference to an element inside the CTA button, not the whole button.
<img width="828" alt="Screen Shot 2021-11-23 at 9 04 14 AM" src="https://user-images.githubusercontent.com/3860311/143038801-08d7a1bd-c066-4fed-8c5e-e8d02668dfe2.png">
I think we we should mention that it's the anchor el. 
It could be something like `ctaButtonAnchorEl`